docs/index.asciidoc

```diff
@@ -109,14 +111,80 @@ append and indirect cannot be combined and will fail validation. +
 `%{&+something}` will add a value to the `+something` key, again probably unintended. +
 ===============================
 
-*Delimiter repetition:* +
-In the source text if a field has variable width padded with delimiters, the padding will be ignored. +
-e.g. for texts of:
+*Behaviour when multiple delimiter are found in the source text:* +
+
+[IMPORTANT]
+===============================
+Starting from version 1.1.1 of this plugin, multiple found delimiter handling has changed.
```

Comment:
Suggest changing this to say: "Starting with version 1.1.1 of this plugin, delimiter handling has changed. Now multiple consecutive delimiters will be seen as empty fields by default and not padding."